### PR TITLE
Update reading-time to include content collections instructions

### PR DESCRIPTION
**Pull request by TheOtterlord**
#### What kind of changes does this PR include?

- New or updated content

#### Description

- What does this PR change? Adds a section describing how to use the remark provided frontmatter with content collections.

### CODE REVIEW

**Review by ElianCodes — APPROVED** [review]
Nice! I think it's a good idea to include this, since content collections are now the way to go!

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

File: src/content/docs/en/recipes/reading-time.mdx [hunk 1]
```diff
@@ -85,4 +85,32 @@ Create a [remark plugin](https://github.com/remarkjs/remark) which adds a readin
       <slot />
     </body>
   </html>
-  ```
\ No newline at end of file
+  ```
+  
+  For [Content Collections](/en/guides/content-collections/), you can access the `remarkPluginFrontmatter` from the `render()` function of a collection entry.
```
Comment (sarah11918) [inline src/content/docs/en/recipes/reading-time.mdx:90]:
```suggestion
 If your blog posts are stored in a [content collection](/en/guides/content-collections/), access the `remarkPluginFrontmatter` from the `entry.render()` function.
```
Don't use "you can" if it really means "you must"!  :smile:

And, I think we also need to  make it clear that these are two different options, and there's no "if"/conditional until you get to this second option.

I think I'd propose:

> 4. If your blog posts are stored in a content collection.... (then all this new stuff here)
>
> To add the remark plugin to pages outside of a collection... (then the old stuff)

What do you think?

**Review by sarah11918 — COMMENTED** [review]
Love the addition @TheOtterlord ! See what you think about my comment below!

File: src/content/docs/en/recipes/reading-time.mdx [hunk 2]
```diff
@@ -71,7 +71,35 @@ Create a [remark plugin](https://github.com/remarkjs/remark) which adds a readin
 
 4. Display Reading Time
 
-  Use the `minutesRead` frontmatter property from `Astro.props` to add the reading time to a [Markdown layout](/en/guides/markdown-content/#markdown-and-mdx-pages). The example layout below displays the reading time above the page content:
+  If your blog posts are stored in a [content collection](/en/guides/content-collections/), access the `remarkPluginFrontmatter` from the `entry.render()` function.
```
Comment (sarah11918) [inline src/content/docs/en/recipes/reading-time.mdx:74]:
```suggestion
  If your blog posts are stored in a [content collection](/en/guides/content-collections/), access the `remarkPluginFrontmatter` from the `entry.render()` function. Then, render `minutesRead` in your template wherever you would like it to appear.
```

**Review by sarah11918 — APPROVED** [review]
I love the flow of this now! Thanks @TheOtterlord !

Two quick additions to actually mentioning *using* the property in the template, and, can you check the indentation on these? The original page seems off, too, but these steps should end up indented to be at the same level as the numbered steps. Right now, they're too far left.

After that, good to merge!  :partying_face:

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @TheOtterlord 🙌 feel free to merge it.

**Pull request merged**